packages/components/shared/src/controllers/anchor.ts

```diff
@@ -24,7 +25,8 @@ export class AnchorController implements ReactiveController {
       if (anchorElement) {
         this.#cleanup = positionPopover(host, anchorElement, {
           arrow: this.#config?.arrow,
-          position: this.position ?? this.#config.position ?? 'top'
+          position: this.position ?? this.#config.position ?? 'top',
+          maxWidth: this.maxWidth ?? this.#config.maxWidth ?? undefined
```

Comment:
This feels superfluous. If `this.maxWidth` is `null` or `undefined`, it will be `undefined`? I think you could just do `maxWidth: this.maxWidth`?